.gitlab/e2e.yml

```diff
@@ -1,6 +1,8 @@
 ---
 # e2e stage
 # Contains jobs which runs e2e tests on our Docker images.
+.if_e2e: &if_e2e
```

Comment:
I am wondering if we could create a rule in `.gitlab-ci.yaml` that could be used everywhere with `!reference` so we do not need to replicate this part. WDYT?
I think it would be helpful too if we decide to split e2e test in several files when we have a lot of them.

I am not 100% sure we can actually put several `!reference` in one `rules` list. If you already tried we can let it as is

Reply:
👍 addressed in 452be97305